app/models/attachment.rb

```diff
@@ -115,15 +113,20 @@ def prepared?
 
   # images are sent inline
   def inlineable?
-    is_plain_text? || is_image? || is_pdf?
+    is_plain_text? || is_image? || is_movie? || is_pdf?
   end
 
+  # rubocop:disable Naming/PredicateName
   def is_plain_text?
-    ALLOWED_TEXT_TYPES.include?(content_type)
+    OpenProject::MimeType::PLAIN_TEXT_TYPES.include?(content_type)
```

Comment:
It's not critical but IMHO this could be put into the methods `movie?`, `plain_text?` etc. directly in the `OpenProject::MimeTypes` module/class and these calls could be delegated to that.

Reply:
Sure 👍